src/streamSaver.js

```diff
@@ -3,8 +3,9 @@ const { exec } = require('child_process');
 const logger = require('./logger');
 
 const browserPath = '/usr/bin/google-chrome';
+//const browserPath = 'C:\\\\\\\\\\\\\\\\Program Files\\\\\\\\\\\\\\\\Google\\\\\\\\\\\\\\\\Chrome\\\\\\\\\\\\\\\\Application\\\\\\\\\\\\\\\\chrome.exe';
```

Comment:
Maybe this will be useful.
'C:\\Program Files\\Google\\Chrome\\Application\\chrome.exe';